modules/apps/portal-search/portal-search-rest-impl/src/main/java/com/liferay/portal/search/rest/internal/odata/entity/v1_0/SearchResultEntityModel.java

```diff
@@ -39,14 +39,31 @@ public SearchResultEntityModel() {
 				"dateCreated",
 				locale -> Field.getSortableFieldName(Field.CREATE_DATE),
 				locale -> Field.CREATE_DATE),
+			new DateTimeEntityField(
+				"dateDisplay",
```

Comment:
this is a question.

Do you know why on this DateTimeEntityField names are "date*" and not "*Date" ??

Reply:
AFAIK, it is the convention for headless. No other reason.